src/mockSdk.js

```diff
@@ -14,7 +14,7 @@ export const mockSDKPerson = {
   orgId: 'orgID',
 };
 
-export const mockSDKMeeting = {
+export const createMockSDKMeeting = () => ({
```

Comment:
jsdoc. And better use the "function createMockSDKMeeting" syntax instead of arrow.